src/util.js

```diff
@@ -13,3 +13,13 @@ export const objectFromArray = (arr, createValue) =>
 export const mapObject = (fn, obj) => Object.keys(obj).reduce(
   (acc, key) => ({ ...acc, [key]: fn(obj[key], key, obj) })
 , {});
+
```

Comment:
just knowing when you import files from here, they won't mess with anything outside.